common/lib/xmodule/xmodule/modulestore/split_mongo/mongo_connection.py

```diff
@@ -311,14 +315,19 @@ def get_structure(self, key, course_context=None):
         with TIMER.timer("get_structure", course_context) as tagger_get_structure:
             cache = CourseStructureCache()
 
-            structure = cache.get(key)
-            tagger_get_structure.tag(from_cache='true' if structure else 'false')
+            structure = cache.get(key, course_context)
+            tagger_get_structure.tag(from_cache=str(bool(structure)))
```

Comment:
We currently have true/false, this will introduce True/False -- can we keep it lowercase for consistency?